### ISSUE TITLE
How to load this solution in Visual Studio?

### ISSUE THREAD

**Issue by elhaix** [Genymobile/scrcpy#1004]
Hey there and thank you for the great application!

There is a scrcpy wrapper [Visual Studio Extension](https://marketplace.visualstudio.com/items?itemName=pvdstel.scrcpy) ([source code](https://github.com/pvdstel/scrcpy-vs/)) (which I haven't gotten to work yet), that I was looking at in hopes that it would expose scrcpy functionality to allow me to begin extending its features in C#.

However, it seems this is not the case and am looking for a way to do this.  

I have tried creating a C++ project in Visual Studio and importing the files.  No go.

Suggestions and assistance appreciated.

**Comment by rtrahms** [comment]
I agree, this application is super cool!  We also are interested in integration of this with some other client processing using a VS toolchain and are curious on the best way to do that.

**Comment by chenwc520** [comment]
Hope to have a general SDK

**Comment by rom1v** [comment]
Sorry, I have no idea.
